crates/autopilot/src/infra/solvers/dto/solve.rs

```diff
@@ -103,6 +103,7 @@ pub struct Solution {
     pub orders: HashMap<boundary::OrderUid, TradedAmounts>,
     #[serde_as(as = "HashMap<_, HexOrDecimalU256>")]
     pub clearing_prices: HashMap<H160, U256>,
+    pub gas: Option<u64>,
```

Comment:
Here doesn't have to be optional right?

Reply:
Or do we want to have ALL quotes send to autopilot and there decide how to handle them

Reply:
Since there are circumstances where we are able to verify a quote making this optional seems reasonable to me. If the quoter wants to make sure their quote will definitely be used they should return a gas estimate but if they don't care then we can simply decide in the autopilot how we want to handle this.
Also technically even if we make this required a solver could still just make up some number and circumvent the issue that way.